app/models/recognition.js

```diff
@@ -11,7 +11,7 @@ export default class RecognitionModel extends Model {
   association;
   @belongsTo('period', {
     inverse: null,
-    async: true,
+    async: false,
```

Comment:
This makes code a lot easier to manage since we can access the relationship in a sync way and we are already using includes everywhere anyway.